### DISCUSSION TITLE
Select/option and Input value in one array

### DISCUSSION THREAD

**Discussion by FreytagDom** [neuefische/questions#180] [Web] [answered]
Hallo zusammen,
ich stehe grade vor dem Problem das ich nicht weiter komme. Ich möchte aus einem dropDown menu (select/option) einen Punkt auswählen und diesem etwas aus den 'input' Feldern mitgeben. Mein Problem ist das ich es nicht schaffe den ausgewählten Punkt des dropDown mit den Daten aus dem input zu verknüpfen. Am Ende sollen sie auf einer anderen Seite gerendert werden.
Wie man sehen kann habe ich schon einige Sachen ausprobiert.
Ich hoffe mir kann hier einer helfen oder einen Tipp geben.

Danke schon einmal :) 

![Bildschirmfoto 2022-09-28 um 15 53 21](https://user-images.githubusercontent.com/109357762/192799100-52fd198b-6659-4b14-bc3a-c12785029365.png)
![Bildschirmfoto 2022-09-28 um 15 53 35](https://user-images.githubusercontent.com/109357762/192799133-51f4f351-cdfc-49ed-a7b3-0e112c1bf21c.png)

![Bildschirmfoto 2022-09-28 um 15 53 45](https://user-images.githubusercontent.com/109357762/192799172-9f67122c-48e3-4b03-a0e0-976ebbd203ba.png)

**Comment by NikRoe** [comment]
Hey Dominic,
can you send us a Pull Request? 

Thanks in advance :)

**Reply by FreytagDom** [reply]
Hey Niklas.
https://github.com/FreytagDom/capstone-project/pull/9

**Comment by FreytagDom** [comment, marked as answer]
Ich habe das Problem nun doch lösen können.

setInsulinFactor:  
Zeile 34 :  ```const insulin = form.insulinSelect.value; ```

in Zeile 44 neu : ```const options = [
    { value: 'basal', label: 'Basal' },
    { value: 'bolus', label: 'Bolus' },
    { value: 'fiasp', label: 'Fiasp' },
  ]; ```

zum Schluß  ```<InsulinSelect /> ersetzt durch : 
  <InsulinSelect
              htmlFor="insulin"
              name="insulinSelect"
              id={InsulinOption.id}
            >
              {options.map((option) => (
                <InsulinOption
                  value={option.value} >
                  {option.label}
              </InsulinOption>
              ))}
            </InsulinSelect>``` 
ersetzt.

Somit ist mein Problem gelöst!!
